shared/profile/dumb.desktop.js

```diff
@@ -128,13 +129,17 @@ const propsBase: RenderProps = {
   following,
   trackerState: normal,
   currentlyFollowing: false,
+  onPushProfile: username => console.log('onpush', username),
+  onBack: () => console.log('onBack'),
+  refresh: () => console.log('refresh'),
   onFollow: () => console.log('onFollow'),
   onUnfollow: () => console.log('onUnfollow'),
   onAcceptProofs: () => console.log('onAcceptProofs'),
   onFolderClick: folder => { console.log('onFolderClick', folder) },
   onUserClick: username => { console.log('onUserClick', username) },
   parentProps: {
     style: {
+      ...globalStyles.flexBoxColumn,
```

Comment:
it wasn't working before (i was getting overflow) but seems to work now with the recent changes to dumb sheet